server/routers/jobs_router.py

```
@@ -1,145 +1,270 @@
+            data = {"resultats": []}
+        
+        if len(data.get("resultats", [])) == 0:
+            break
+
+        resultat += data.get("resultats", [])
+        page += 1
+
+    return resultat
+
+def calcul_salaire(texte_salaire: str, texte_heure: str) -> float:
+    if texte_salaire == None:
```

Ruff (0.14.13)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)